src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -8526,7 +8527,7 @@ void CmdSketcherCompConstrainRadDia::languageChange()
     QAction* arc3 = a[2];
     arc3->setText(QApplication::translate("CmdSketcherCompConstrainRadDia",
```

Comment:
is `CmdSketcherCompConstrainRadDia` a typoe as well ?

Reply:
No, it's an abbreviation for **Rad**ius and **Dia**meter